src/main/cpp/src/parse_uri.cu

```diff
@@ -567,17 +581,17 @@ uri_parts __device__ validate_uri(const char* str, int len)
       if (next_slash == -1 && ret.authority.size_bytes() == 0 && ret.query.size_bytes() == 0 &&
           ret.fragment.size_bytes() == 0) {
         // invalid! - but spark like to return things as long as you don't have illegal characters
-        // ret.valid = false;
-        ret.valid = true;
+        // ret.valid = 0;
```

Comment:
Uncomment it?

Reply:
I specifically left it commented with the comment above explaining why. The RFC says this is not valid, but Java/Spark disagrees.